src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,102 @@
+package duke;
+
+/**
+ * Class that is a chat bot that can store tasks.
+ */
+public class Duke {
+    private boolean isRunning = true;
+    private enum Commands { list, done, todo, event, deadline, delete, find }
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Class Constructor based on where the storage file path.
+     *
+     * @param filePath where the storage file is at.
+     */
+    public Duke (String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.parseFile());
+        } catch (Exception e) {
```

Comment:
Pokemon exception handling (Catching them all) is generally not recommended. May I suggest listing and catching specific intended exceptions? 

eg) catch (ParseException | IllegalArgumentException | FileNotFoundException e) {...}